code/modules/mob/living/simple_animal/hostile/syndicate_mobs.dm

```diff
@@ -34,16 +34,17 @@
 	sentience_type = SENTIENCE_OTHER
 	footstep_type = FOOTSTEP_MOB_SHOE
 	robust_searching = TRUE
+	var/list/loot_override = list(/obj/effect/mob_spawn/human/corpse/syndicatesoldier,
```

Comment:
Please do not forget to audodoc new vars

Reply:
Unnecessary now, the var doesn't exist anymore.